backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/callout/C_Order.java

```diff
@@ -81,16 +82,10 @@ public void setDropShipWarehouse(final I_C_Order order, final ICalloutField fiel
 		}
 	}
 
-	@CalloutMethod(columnNames = {
-			I_C_Order.COLUMNNAME_C_BPartner_ID,
-			I_C_Order.COLUMNNAME_C_BPartner_Location_ID,
-			I_C_Order.COLUMNNAME_AD_User_ID },
-			skipIfCopying = true)
+	@CalloutMethod(columnNames = I_C_Order.COLUMNNAME_AD_User_ID, skipIfCopying = true)
 	public void updateBPartnerAddress(final I_C_Order order)
 	{
-		documentLocationBL.updateRenderedAddressAndCapturedLocation(OrderDocumentLocationAdapterFactory.locationAdapter(order));
-		
-		orderBL.setM_PricingSystem_ID(order, true); // changed partner/location might imply a changed PS/PL
+		updateRenderedAddressCapturedLocationAndPricingSystem(order);
```

Comment:
Consider introducing documentLocationBL.updateRenderedAddress AFAIS this is the only relevant part here